.eslintrc.cjs

```diff
@@ -36,6 +36,7 @@ module.exports = {
     ],
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-inferrable-types': 'off',
+    '@typescript-eslint/no-unused-vars': ['error', { ignoreRestSiblings: true, argsIgnorePattern: '^_' }],
```

Comment:
What is this one for?

Reply:
This basically let's us write code like

``` const { foo: _ignore, ...restOfStuff } = bar```

and eslint won't complain about `_ignore` not being used if we intend to delete it from the object.

Reply:
Ah i see, cool.